doc/source/yt4differences.rst

```diff
@@ -159,24 +157,20 @@ It should be noted that the
     # generate an octree
     octree = ds.octree(left, right, n_ref=64)
 
+    ds.sph_smoothing_style = "scatter"
+
     # the density will be calculated using SPH scatter
     density = octree[('PartType0', 'density')]
 
     # this will return the x positions of the octs
     x = octree[('index', 'x')]
 
 The above code can be modified to use the scatter approach by using
-``ds.sph_smoothing_style = 'gather'`` before any field access. The octree also
-accepts ``over_refine_factor`` which works just like the ``over_refine_factor``
-parameter in yt-3.0 that could be passed to ``yt.load``, and determines how many
-particles are in each leaf.
-
-The ``density_factor`` keyword allows the construction of dense octrees
-trees. In a traditional octree, if a leaf has more particles that a critical
-value `n_ref`, then it divides into 8 new children (hence the name oct). The
-value of `density_factor` allows the node to divide into 2^(3*density_factor)
-zones instead. This creates an octree structure similar that used by AMR codes
-like FLASH that make use of an octree of grid patches.
+``ds.sph_smoothing_style = 'gather'`` before any field access.
+
+The octree implementation is very simple. It uses a recursive algorithm to build
+a ``depth-first`` which is consistent with the results from yt-3.
```

Comment:
We don't have documentation anywhere describing a depth-first octree, right? This isn't my area, but it might be worth adding a sentence explaining what this is.

Reply:
The explanation is a little abstract, but hopefully that is more informative than before